src/Stick.js

```diff
@@ -22,28 +21,36 @@ import { type AlignT, type ApiPropsT, type PositionT } from './flowTypes'
 import { useAutoFlip, useWatcher } from './hooks'
 import { getDefaultAlign, getModifiers, scrollX, uniqueId } from './utils'
 
-type PropsT = {|
-  ...ApiPropsT,
-
-  style: Substyle,
-|}
-
-function Stick({
-  inline,
-  node,
-  style,
-  sameWidth,
-  children,
-  updateOnAnimationFrame,
-  position,
-  align,
-  component,
-  transportTo,
-  autoFlipHorizontally,
-  autoFlipVertically,
-  onClickOutside,
-  ...rest
-}: PropsT) {
+const useStyle = createUseStyle(
+  {
+    node: {
+      position: 'absolute',
+      zIndex: 99,
+      textAlign: 'left',
+    },
+  },
+  getModifiers
+)
+
+function Stick(props: ApiPropsT) {
+  const style = useStyle((props: Object))
```

Comment:
This cast shows that there is something wrong with how `useStyle` is typed. This shouldn't be necessary. I also find it a bit weird, how this new API now makes it essentially impossible to destructure the props parameter. 

I'd have expected a syntax like: 

```
const style = useStyle(modifiers)

// or

const style = useStyle(styles, modifiers)
```

Which would require much less rewriting. What is the benefit of the `createUseStyles` hooks creator method?

Reply:
Does this have something to do with allowing style overwrites? Could this also be more explicit by including it in the API? 

```
const style = useStyle(styles, modifiers, props.style)
```